### ISSUE TITLE
Update OE ONRR Enforcement Updates (02/2021) document to accessible version

### ISSUE THREAD

**Issue by lpgoldstein** [ONRR/onrr.gov-site#1766]
Current version on site is not accessible. Update reading order on several slides and reupload.
[OE-STRAC-Presentation-2021-accessibilitycheck.pptx](https://github.com/ONRR/onrr.gov-site/files/8374060/OE-STRAC-Presentation-2021-accessibilitycheck.pptx)

**Comment by lpgoldstein** [comment]
@mcharg ready for review http://2017onrr/compliance/civil-penalties.htm

**Comment by mcharg** [comment]
@lpgoldstein Looks good.

**Comment by lpgoldstein** [comment]
live so closing
